drakrun/drakrun/draksetup.py

```diff
@@ -551,19 +554,20 @@ def postinstall(report, generate_usermode):
         f.write(runtime_info.to_json(indent=4))
 
     logging.info("Saving VM snapshot...")
-    subprocess.check_output('xl save vm-0 ' + os.path.join(VOLUME_DIR, "snapshot.sav"), shell=True)
+    subprocess.check_output('xl save -c vm-0 ' + os.path.join(VOLUME_DIR, "snapshot.sav"), shell=True)
```

Comment:
How about adding a new method to `VirtualMachine` class? For example:
`def save(self, file_path: str, destroy=True)`

Reply:
Same with this change also(regarding different PR) if it seems fine

Reply:
:+1: